platform/andes/ae350/config.mk

```diff
@@ -0,0 +1,36 @@
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# Copyright (c) 2019 Western Digital Corporation or its affiliates.
```

Comment:
Please change this Copyright. It should be copyright by AndesTech right?

Reply:
Yes, thank you for your reminder.